### ISSUE TITLE
Transmission BT connectivity now an issue

### ISSUE THREAD

**Issue by Infallibleman** [SchizoDuckie/DuckieTV#1404]
**What build of DuckieTV are you using (Standlone / Chrome Extension (New Tab / Browser Action))**
Standalone 202311011754

**What version of DuckieTV are you using (Stable 1.1.x / Nightly yyyymmddHHMM)**
Nightly 202311011754

**What is your Operating System (Windows, MacOSx, Linux)**
Mac os 10_15_7

**Describe the problem you are having and steps to reproduce if available**
I have transmission BT which worked great with duckie tv. now when i start Duckie tv, it is stuck at "connecting to transmission, please wait"

**Attach any DuckieTV statistics or Developer Console logs if available**
Not sure where to get these from?

**Comment by garfield69** [comment]
What version of Transmission BT are you on?
Can you click on the About icon near the bottom left of the Dtv main page, to open the right side panel, scroll down to the Statistics section, click on the statistics header, which should copy it all to your clipboard.
Then paste the content to a plain text editor file and save as a stats.txt and drag/drop the file in a new comment box here.
If you click on the Dtv Settings>Transmission Integration button, what does the right sidepanel show for status?
When you use Cmd-Alt-I to open the Development tools page, and click on the Console tab to view the log, do you see messages related to connecting to transmission?
Thanks.

**Comment by garfield69** [comment]
Also, did you recently upgrade your Transmission BT, if so, what was the version before your current one?
I do not have a Mac OsX machine so i cannot do a parallel test, but on my win10 I upgraded from TransmissionBT v3 to Transmission BT v4.0.5 just now.
I found that my development DuckieTV did not connect to the new Transmission BT.
So first I used my browser to check if I could start a session with Transmission BT by using 127.0.0.1:9091 and discovered that the username and password were not being accepted.
So I shutdown Dtv, and accessed the Transmission BT app and navigated to the settings>Remote page, and replaced my password and saved.
I quit the app and started it afresh, then I tested the browser again to confirm my credentials which now worked.
Then I started Dtv and it was able to connect.
Maybe this is relevant to you too ;-)
